src/WorkItemMigrator/Migration.WIContract/WiAttachment.cs

```diff
@@ -1,15 +1,28 @@
 ﻿using System.IO;
-using System;
 
 namespace Migration.WIContract
 {
     public class WiAttachment
     {
         public ReferenceChangeType Change { get; set; }
-        public string FilePath { get; set; }
+        public string FileName { get; private set; }
         public string Comment { get; set; }
         public string AttOriginId { get; set; }
 
+        private string filePath;
+        public string FilePath
+        {
+            get
+            {
+                return filePath;
+            }
+            set
+            {
+                filePath = value;
+                FileName = Path.GetFileName(value);
+            }
+        }
+
         public override string ToString()
         {
             return $"[{Change.ToString()}] {AttOriginId}/{Path.GetFileName(FilePath)}";
```

Comment:
YOu can use FileName instead of FilePath here?

Reply:
Thanks, will fix